package.json

```diff
@@ -122,6 +123,7 @@
     "i18next": "^19.6.3",
     "immutability-helper": "2.6.4",
     "interweave": "^11.1.0",
+    "levelup": "^4.4.0",
```

Comment:
What's this is used for?

Reply:
work-around for a 3box bug.  before submitting this for review (note it is still a draft) I plan to check whether they have fixed the bug

Reply:
I have a query in with 3box to discover whether this work-around is still necessary